automatic/PDFCreator/PDFCreator.nuspec

```diff
@@ -17,8 +17,4 @@ PDFCreator – PDFCreator easily creates PDFs from any Windows program. Use it l
     <tags>pdf admin</tags>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
```

Comment:
This migration to AU is perfect time to tidy up metadata. You need to revisit everything anyway, and those information are really handy to have in `choco info <package>` or on site directly.